pkg/github/config.go

```diff
@@ -75,13 +77,21 @@ func (c *Config) RegisterFlags(set *cli.FlagSet) {
 		if err := json.Unmarshal(data, &event); err == nil {
 			d.PullRequestNumber = event.GetNumber()
 			d.PullRequestBody = event.GetPullRequest().GetBody()
+			d.Actor = event.GetSender().GetLogin()
 		}
 	}
 	if githubContext.EventName == "pull_request_target" {
 		var event github.PullRequestTargetEvent
 		if err := json.Unmarshal(data, &event); err == nil {
 			d.PullRequestNumber = event.GetNumber()
 			d.PullRequestBody = event.GetPullRequest().GetBody()
+			d.Actor = event.GetSender().GetLogin()
+		}
+	}
+	if githubContext.EventName == "workflow_dispatch" {
+		var event github.WorkflowDispatchEvent
+		if err := json.Unmarshal(data, &event); err == nil {
+			d.Actor = event.GetSender().GetLogin()
```

Comment:
I think this is automatically injected into the environment, is there a difference or does it differ in certain scenarios where we would want sender vs actor?

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables:~:text=GITHUB_ACTOR,example%2C%20octocat.

Reply:
It looks like GitHub sender is guaranteed to be a user, but actor can be both a user or an app. Not sure if it matters, so using actor for now

Reply:
sgtm